src/custom-elements/src/components/date-picker/date-picker.tsx

```diff
@@ -338,27 +292,78 @@ export class DatePicker {
     }, CHI_TIME_AUTO_SCROLL_DELAY);
   }
 
+  @Listen('chiDateChange')
+  handleDateChange(ev: CustomEvent) {
+    const target = ev.target as HTMLElement;
+
+    if (target.nodeName === 'CHI-DATE-PICKER') {
```

Comment:
Is it always uppercase or would it be safer to check  `target.nodeName.toUpperCase()` just in case?

Reply:
Done.